src/ShopPaymentIPNBundle/objects/CronJobs/TPkgShopPaymentIPN_TCmsCronJob_ProcessTrigger.class.php

```diff
@@ -11,6 +11,13 @@
 
 class TPkgShopPaymentIPN_TCmsCronJob_ProcessTrigger extends TdbCmsCronjobs
 {
+
+
+    public function __construct()
```

Comment:
No empty constructors needed.

(And no empty lines at the start of the class.)

Reply:
removed.